### ISSUE TITLE
Add preprocessing to crop images

### ISSUE THREAD

**Issue by sumiya-kuroda** [neuroinformatics-unit/photon-mosaic#24]
**Is your feature request related to a problem? Please describe.**
Two-photon imaging of the mouse brain while showing visual stimuli often ends up having light contamination at the image edges.

**Describe the solution you'd like**
Add image cropping to the preprocessing function libraries.

**Comment by lauraporta** [comment]
You're rising an interesting problem. We can definitely add such a function in the preprocessing. 
I wonder, how many pixels would you discard? Is it an artefact you would not observe if you were to image in darkness?
Would you mind including an example of the artifact?

**Comment by sumiya-kuroda** [comment]
When we present visual stimuli to animals, we typically blank the monitor with custom electronics to ensure the monitor is turned off during imaging acquisition. Here is how we do it in SWC: https://github.com/SainsburyWellcomeCentre/monitor_blanking

This will typically end up having a monitor light contamination at the edges of the X-axis. e.g. the yellow stripe patterns in this example mean image. (Basically monitor light is way, way stronger than the  gcamp signal)
<img width="1033" height="986" alt="Image" src="https://github.com/user-attachments/assets/935303df-44b1-46b8-a015-6286f2e7ef4e" />

Here is the Python function I was using:
```py
def save_trimmed_tif(tif_path, output_path, output_path_chan2=None, trimpix=0, tif_shape=(512, 512, 3000), chunk_size=1000,
                     clusterize=False):
    if not clusterize:
        lazy_imread = delayed(imread) # set up lazy image reader
        reader = lazy_imread(tif_path)  # doesn't actually read the file

        dask_img = da.from_delayed(reader, shape=(tif_shape[2], tif_shape[0], tif_shape[1]), dtype='int16')

        with tqdm(total=dask_img.shape[0], desc="Saving trimmed frames as tif") as pbar:
            for start_frame in range(0, dask_img.shape[0], chunk_size):
                end_frame = min(start_frame + chunk_size, dask_img.shape[0])
                
                # Compute the chunk (slice of the Dask array)
                chunk = dask_img[start_frame:end_frame, trimpix:-trimpix, trimpix:-trimpix].compute()

                # Write the chunk to the TIFF file
                if start_frame == 0:
                    # For the first chunk, create the TIFF file
                    tifffile.imwrite(output_path, chunk, bigtiff=True)
                else:
                    # For subsequent chunks, append to the TIFF file
                    tifffile.imwrite(output_path, chunk, append=True, bigtiff=True)

                pbar.update(chunk_size)  # Update progress bar
    else: # running on cluster?
        # img_np = tifffile.imread(tif_path)
        img_np = ScanImageTiffReader(tif_path).data()
        # img_jnp = jnp.array(img_np)
        trimmed_np = img_np[:, trimpix:-trimpix, trimpix:-trimpix]
        if output_path_chan2==None:
            tifffile.imwrite(output_path, trimmed_np, bigtiff=True)
        else: # chan2 exists
            indices_chan1 = np.arange(img_np.shape[0])
            mask_chan1 = indices_chan1 % 2 == 0 # TODO: find robust method

            tifffile.imwrite(output_path, trimmed_np[mask_chan1], bigtiff=True)   
            tifffile.imwrite(output_path_chan2, trimmed_np[~mask_chan1], bigtiff=True)          

    return None
```

**Comment by lauraporta** [comment]
Thank you @sumiya-kuroda. We can definitely include this feature.
